### DISCUSSION TITLE
require.context errors out with "context" is unresolved function

### DISCUSSION THREAD

**Discussion by ponienchen** [webpack/webpack#13521] [Q&A]
I have both webpack and webpack-cli installed, and I would like to use require.context, but intelliJ complains about "context" being unresolved.

Did I miss a step somewhere ?
Do I need to install another npm package or add something somewhere to make it work ?

Could someone please help me out?
Thanks a lot!

**Comment by alexander-akait** [comment]
Please ask this in `intelliJ` company, it is IDE problem
